integration/fuse/src/main/java/alluxio/fuse/AlluxioFuse.java

```diff
@@ -171,6 +175,10 @@ public static void launchFuse(FileSystemContext fsContext,
       }
     } catch (Throwable e) {
       throw new IOException("Failed to mount Alluxio file system", e);
+    } finally {
+      if (blocking) {
```

Comment:
If blocking is false, i previously rely on process kill to close fs, now change to close fs in FuseManager.close() so that in the future, we can have more control of the embedded fuse manager